src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -220,47 +212,56 @@ public void clickAddSectionLevelPrivilegesLink(int instrNum) {
         this.addSectionLevelPrivilegesLink(instrNum).click();
     }
     
-    private WebElement sectionCheckBoxInSectionLevel(int instrNum, int sectionLevelNum, int sectionNum) {
-        String cssSelector = "#tuneSectionPermissionsDiv" + sectionLevelNum + "ForInstructor" + instrNum
-                + " input[name=sectiongroup" + sectionLevelNum + "section" + sectionNum + "]";
-        return browser.driver.findElement(By.cssSelector(cssSelector));
+    public WebElement sectionLevelIndexPanel(int instrNum, int sectionLevelIndex) {
+        String permissionDivId = "tuneSectionPermissionsDiv" + sectionLevelIndex + "ForInstructor" + instrNum;
+        return browser.driver.findElement(By.id(permissionDivId));
+    }
+    
+    private WebElement sectionCheckBoxInSectionLevel(int instrNum, int sectionLevelIndex, int sectionNum) {
```

Comment:
Maybe rename it `sectionSelectionCheckBox`, since this checkbox only exists in section level panel